explorer/app/config/model.ts

```diff
@@ -25,6 +26,16 @@ export interface EntityConfig<
   staticLoad?: boolean;
 }
 
+export interface ComponentObject<
+  T extends keyof JSX.IntrinsicElements | JSXElementConstructor<any> = any,
+  D = any
+> {
+  component: React.FC<any>;
```

Comment:
Super interesting, let's talk through this on our call this afternoon. We're curious to understand in more detail how this works.